literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi: 10.1021/acs.jpcc.8b05685 # only the number without preceeding doix or https
+    bib: mello_2018_bromide_18562 # AuthorName_YYYY_FirstWordTitle_pageNr. No prepositions. Fingers crossed that there is no duplicate.
+    figure: 1 # depending on the subfigure labels use 1, 1b, or 1_3
+    curve: ph1,2_black # use a unique short description to identify the curve in the figure, i.e.. label, color, etc
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-12-01 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
+    comment: Working solutions of the pH-series with values lower than 3 were prepared by using HClO4 and KClO4. For pH values ranging from 3 to 9, phosphate buﬀer solutions were employed (Na2HPO4 and NaH2PO4). Alkaline solutions were prepared with NaOH and KClO4. The exact concentrations of these components are unkown. # Add whatever you think is important to understand the data
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: HClO4 # can be trivia name, sum formula, etc
+              concentration:
+                  value:
+                  unit: #[M, mM, µM, g kg-1, ...]
+              type: acid #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: KClO4
+              type: salt
+              concentration:
+                  value: 
+                  unit:
+            - name: 
+              type: 
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value:
+                  unit: #[M, mM, µM, g kg-1, ...] 

```

Comment:
What is the meaning of this section?

Reply:
The components dissolved in the solvent.
In the digitization process, I fill out what is given in the paper and do not delete all empty entries.